app/forms/trn_submission_form.rb

```diff
@@ -49,6 +48,10 @@ def funding_details_collected?
     FeatureService.enabled?(:show_funding)
   end
 
+  def should_validate_degree?
```

Comment:
```suggestion
  def validate_degree?
```

Reply:
i knew this was 💩  when i wrote it - many thanks for holding me to account!!